app/frontend/wallet/shelley/shelley-transaction.ts

```diff
@@ -30,35 +52,57 @@ function ShelleyTxAux(inputs, outputs, fee, ttl, certs?, withdrawals?) {
     outputs,
     fee,
     ttl,
-    certs,
+    certificates,
     withdrawals,
     encodeCBOR,
   }
 }
 
-function ShelleyFee(fee) {
-  function encodeCBOR(encoder) {
-    return encoder.pushAny(fee)
-  }
+function ShelleyTxInputs(inputs: _Input[]): TxInput[] {
```

Comment:
I would not pollute the general namespace with types/functions meant for translating our higher-level objects like tx inputs/outputs into their low-level versions ready to be serialized and don't serve a purpose outside of tx serialization which is one of the lowest-level concerns of the wallet

I would name this function something like `cborizeTxShelleyInputs` and call the returned type `cborizedTxInput` to make it obvious it's kind of a low-level type. Similarly for other types being "cborized" here